vaxrank/cli.py

```diff
@@ -239,32 +316,20 @@ def main(args_list=None):
     if args_list is None:
         args_list = sys.argv[1:]
 
-    args = arg_parser.parse_args(args_list)
-    logger.info(vars(args))
-
-    if (len(args.output_csv) == 0 and
-            len(args.output_ascii_report) == 0 and
-            len(args.output_html_report) == 0 and
-            len(args.output_pdf_report) == 0):
-        raise ValueError(
-            "Must specify at least one of: --output-csv, "
-            "--output-ascii-report, "
-            "--output-html-report, "
-            "--output-pdf-report")
-
-    template_data = load_template_data(args)
-
-    if args.output_ascii_report:
-        make_ascii_report(
-            template_data=template_data,
-            ascii_report_path=args.output_ascii_report)
+    if "--input-pickle-file" in args_list:
```

Comment:
Not sure what you mean. Would you want to see a separate binary? (That's what I was trying to figure out earlier, ie how you'd want to interact with the vaxrank CLI in the cases of a pickled load)